blog/2020/07/julia-1.5-highlights.md

```diff
@@ -0,0 +1,338 @@
+@def title = "Julia 1.5 Highlights"
+@def authors = "Jeff Bezanson & Stefan Karpinski"
+@def published = "24 July 2020"
+@def rss_pubdate = (2020, 7, 24)
+@def rss = """Julia version 1.5 has been released, featuring many performance improvements and new capabilities."""
+
+
+Julia version 1.5 has been released. Releases are timed and hence not planned around
+specific features, but this time we seem to have gotten lucky: quite a few major
+developments came together to make 1.5 particularly exciting.
+Let's walk through some highlights.
+
+\toc
+
+## All structs can be stack allocated
```

Comment:
I'm going to change this back because it's not completely true --- there are some limits based on size and the number of pointers.

Reply:
The previous title sounded like we were allocating more things rather than less, so maybe come up with something else?